### ISSUE TITLE
False negative for crates.io

### ISSUE THREAD

**Issue by korken89** [mike42/mdcheckr#17]
Hi, I have identified a false negative on the following project: https://github.com/rust-embedded/cortex-m-rt
Would it be possible to fix this? I have no idea why it is failing though as if you copy the link it works fine.

Thanks!

Output:
```
$ mdcheckr README.md                                                                                                                  [12:54:15]
Checking README.md ..
- Link https://crates.io/crates/cortex-m-rt [ FAIL ]
Warning:README.md:1:Broken link 'https://crates.io/crates/cortex-m-rt'
- Link https://crates.io/crates/cortex-m-rt [ FAIL ]
Warning:README.md:1:Broken link 'https://crates.io/crates/cortex-m-rt'
- Link https://github.com/rust-embedded/wg#the-cortex-m-team [ OK ]
- Link https://docs.rs/cortex-m-rt [ OK ]
- Link LICENSE-APACHE [ OK ]
- Link http://www.apache.org/licenses/LICENSE-2.0 [ OK ]
- Link LICENSE-MIT [ OK ]
- Link http://opensource.org/licenses/MIT [ OK ]
- Link CODE_OF_CONDUCT.md [ OK ]
- Link https://github.com/rust-embedded/wg#the-cortex-m-team [ OK ]
- Image https://img.shields.io/crates/v/cortex-m-rt.svg [ OK ]
- Image https://img.shields.io/crates/d/cortex-m-rt.svg [ OK ]
Done, with 2 problems(s) found.
```

**Comment by korken89** [comment]
The issue seems to be in `check_link()`, the `curl` gets a 404 error even though the site exists.

**Comment by mike42** [comment]
I'll close this as fixed, though it is a site issue in my view.

See also comments @ https://github.com/rust-lang/crates.io/issues/788

**Comment by korken89** [comment]
No problem, it was fixed in PR #18 as well - I should have closed this.
